src/java/test/org/apache/zookeeper/server/ZooKeeperServerMainTest.java

```diff
@@ -133,6 +138,62 @@ public void shutdown() {
     }
 
     /**
+     * Test case for https://issues.apache.org/jira/browse/ZOOKEEPER-2247.
+     * Test to verify that even after non recoverable error (error while
+     * writing transaction log) on ZooKeeper service will be available
+     */
+    @Test(timeout = 30000)
+    public void testNonRecoverableError() throws Exception {
```

Comment:
Do we really need this test case given that we have the one in `NonRecoverableErrorTest.java`? This test seems to be contained in the other further down.